src/edu/washington/escience/myria/operator/TipsyFileScan.java

```diff
@@ -322,7 +326,13 @@ protected Schema generateSchema() {
 
   private static InputStream openFileOrUrlInputStream(String filenameOrUrl) throws DbException {
     try {
-      return new URI(filenameOrUrl).toURL().openConnection().getInputStream();
+      URI uri = new URI(filenameOrUrl);
+      if(uri.getScheme() == null)
```

Comment:
spaces around if/catch (we really should have checkstyle running in travis rather than manually reviewing for style errors).